.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = submodules/volto
 	url = https://github.com/plone/volto.git
 	branch = master
+[submodule "submodules/plone.restapi"]
+	path = submodules/plone.restapi
+	url = https://github.com/plone/plone.restapi.git
```

Comment:
Check https://github.com/plone/plone.restapi/pull/1337#issuecomment-1074195352. Then this could be changed to  branch master. 
I' ll continue reviewing afterwards, OK?

Reply:
I need to keep it as `plone6docs` until it is merged into `master`, else it won't build preview on Netlify. I will change it as soon as it is merged.